app/controllers/v0/sessions_controller.rb

```diff
@@ -50,8 +50,9 @@ def user_attributes
         ssn:          attributes['social']&.first&.delete('-'),
         birth_date:   parse_date(attributes['birth_date']&.first),
         uuid:         attributes['uuid']&.first,
+        loa_highest:  attributes['level_of_assurance']&.first,
```

Comment:
👍  I like this too, I'm bringing some of the MVI id attributes to the root but maybe they should just stay grouped in user.mvi (can discuss in user profile meeting)